.gitignore

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
This file is not required. Remove it.

Reply:
It may be required, because in future, this will help people whoever clones to their local and do changes. isn't it?

Reply:
Ideally, I should be adding this to the repo. Didn't expect these `.idea` files. Its okay. I will accept it. Thanks for improving the repo.